autoload/lsp/ui/vim/hover.vim

```diff
@@ -35,8 +35,18 @@ function! s:handle_hover(server, data) abort
     endif
 
     if !empty(a:data['response']['result']) && !empty(a:data['response']['result']['contents'])
-        call lsp#ui#vim#output#preview(a:data['response']['result']['contents'])
-        return
+        for ui in g:lsp_hover_ui
+            if ui == "float"
+                if exists("*nvim_open_win")
+                    call lsp#ui#vim#float#float_open(a:data['response']['result']['contents'])
+                    return
```

Comment:
Seems like we need to use `echo ''` so that the message `Retrieving hover' gets removed for both float and preview.